packages/flutter/lib/src/cupertino/switch.dart

```diff
@@ -120,8 +120,12 @@ class CupertinoSwitch extends StatefulWidget {
     super.key,
     required this.value,
     required this.onChanged,
+    @Deprecated('Please use activeTrackColor')
```

Comment:
The analyzer enforces a specific format in deprecation notices: https://github.com/flutter/flutter/blob/master/docs/contributing/Tree-hygiene.md#deprecations